frontend/public/components/utils/resource-log.jsx

```diff
@@ -44,6 +48,25 @@ const LogControls = ({dropdown, onDownload, toggleFullscreen, isFullscreen, stat
       {dropdown && <div className="co-toolbar__item">{dropdown}</div>}
     </div>
     <div className="co-toolbar__group co-toolbar__group--right">
+      {!_.isEmpty(podLogLinks) && _.map(podLogLinks, (link, i) => {
+        const compiled = _.attempt(() => _.template(link.spec.hrefTemplate));
```

Comment:
If we can't find a way to prevent code execution with `_.template`, we could fall back to...

```js
template
  .replace('${name}', encodeURIComponent(resource.metadata.name))
  .replace('${container}', encodeURIComponent(containerName))
  // ...
```